dist/gmusic.js

```diff
@@ -211,13 +211,16 @@ proto.playback = {
   toggleShuffle: function () { this.playback._shuffleEl.click(); },
 
   getRepeat: function () {
-    var title = this.playback._repeatEl.getAttribute('title').toLowerCase();
-    if (title.indexOf('repeat off') !== -1) {
-      return GMusic.Playback.NO_REPEAT;
-    } else if (title.indexOf('repeating all') !== -1) {
+    // States possible:
+    // no-repeat: No class 'active', icon: av:repeat
+    // repeat: Has class 'active', icon: av:repeat
+    // repeat-one: Has class 'active', icon: av:repeat-one
+    if (this.playback._repeatEl.__data__.icon === 'av:repeat-one') {
```

Comment:
Yeah makes sense, I've never been good with good documentation. Yours is much cleaner. I'll add it.